nifi-nar-bundles/nifi-deeplearning4j-bundle/nifi-deeplearning4j-processors/src/test/resources/classification_test.txt

```diff
@@ -0,0 +1,100 @@
+1.1,0.5,0.5,0.2,0
```

Comment:
Where did this text file come from? Was it generated by you, or did you find it online somewhere, etc.? Do we have the appropriate license for this?

Reply:
I've mentioned at the beginning of the tests that these are based on deeplearning4j examples.  I generated this very simple/small sample file to make consistent predictions for the tests even with just a few observations.

Reply:
It seems this text and test are based on the [DL4J CSVExample](https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/src/main/java/org/deeplearning4j/examples/dataexamples/CSVExample.java).

IMHO, instead of mock classification, using the [iris.txt](https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/src/main/resources/iris.txt) and the same configuration with CSVExample to train the test model would be more understandable, especially for those new to Deep Learning and Neural Network topic.

Because the iris.txt dataset has more background information what the data is, and what classes the model is supposed to predicate. Current mock classification can not imagine such actual objective and can be hard to understand what it does. And having the same number (4) for different things, such as inputNumber, outputNumber, numClasses makes even harder to grasp what those numbers are.

How do you think?

Reply:
@ijokarumawak - The tests included are to show how we can generate a model and configure the processor for regression and classification in the integration tests.  My thought while creating the mock data was create a the model with reproducible results even with limited observations and few iterations.  In real life, a multilayer model will be created, tested and validated by the user prior to plugging it into the component.

Reply:
@mans2singh Yeah, I understand that. Those test code were helpful to create a model file to use in a NiFi flow. My intention was to make the test data and test case more understandable.